src/main/java/pro/beam/api/resource/chat/methods/WhisperMethod.java

```diff
@@ -0,0 +1,46 @@
+package pro.beam.api.resource.chat.methods;
+
+import com.google.common.collect.ImmutableList;
+import pro.beam.api.resource.BeamUser;
+import pro.beam.api.resource.chat.AbstractChatMethod;
+
+import java.util.List;
+import java.util.Random;
+
+public class WhisperMethod extends AbstractChatMethod {
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        protected BeamUser to;
```

Comment:
Any objections to changing this to `targetUser` to make it more apparent what we need the user for?